source/core/icosahedron.ts

```diff
@@ -0,0 +1,212 @@
+
+import { vec2, vec3 } from 'gl-matrix';
+import { v3 } from '../gl-matrix-extensions';
+
+type ivec3 = [number, number, number];
+
+/**
+ * Helper class to generate a sphere geometry based on a refinable icosahedron
+ */
+export class Icosahedron {
+
+    protected _faces: Array<ivec3>;     // List of faces (triangles)
```

Comment:
This comment style is not used (should not be used) in webgl-operate. Prefer 
```
/**
 * List of faces (triangles).
 */
```
Same applies to all other occurrences within this PR.

Reply:
If there is a corresponding getter/setter then use:
```
    /** @see {@link faces} */
    protected _faces: Array<ivec3>;
```